tests/test_matching.py

```diff
@@ -252,37 +295,68 @@ def test_setup_matching_with_tpms(self, client, test_client, helpers):
         conference = builder.get_result()
         assert conference, 'conference is None'
 
-        ## Set up matching
-        conference.setup_matching(tpms_score_file= os.path.join(os.path.dirname(__file__), 'data/tpms_scores.csv'))
-
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Assignment_Configuration')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Custom_Load')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Conflict')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Aggregate_Score')
-        assert client.get_invitation(id = 'auai.org/UAI/2019/Conference/-/Reviewing/Paper_Assignment')
+        # Set up reviewer matching
+        conference.setup_matching(
+            conference.client.get_group(conference.get_reviewers_id()),
+            tpms_score_file=os.path.join(os.path.dirname(__file__), 'data/reviewer_tpms_scores.csv'))
+
+        print(conference.get_reviewers_id())
+        reviewer_matching_ids = _matching_invitation_ids(
+            client, conference.get_reviewers_id())
+
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Assignment_Configuration')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Custom_Load')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Conflict')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Aggregate_Score')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/Paper_Assignment')
+        assert client.get_invitation(id='auai.org/UAI/2019/Conference/Program_Committee/-/Reviewing/TPMS_Score')
```

Comment:
do you think it is better to have an affinity/tpms score invitation per matching group?

I'm realizing the bid invitation is being used to both groups: area chairs and reviewers and the invitation.reply.content is something like this:

```
                'content': {
                    'head': {
                        'type': 'Note'
                    },
                    'tail': {
                        'type': 'Group'
                    },
                    'label': {
                        'value-radio': ['Very High', 'High', 'Neutral', 'Low', 'Very Low'],
                        'required': True
                    }
                }
```

where the tail can be any group, do we need to split the bid invitation too? or should we keep using one invitation for all matching groups? 

@zbialecki thoughts?

Reply:
I don't have an opinion about bids. But I think it's slightly simpler to have TPMS/affinity scores be separated by matching group, just because of the way scores get generated. Our expertise modeling framework is based on groups, and by default, it generates one score file per group. But that can be changed, if there's a good reason to keep them all together.

Reply:
I'm OK with having a separate affinity score invitation for each matching group, but it will add some inefficiencies because for example Area Chairs and Reviewers overlap so much. So we'll have a bunch of duplicated edges.

Reply:
Do ACs and Reviewers typically overlap? if I remember correctly, ICLR doesn't do that, but maybe other conferences do

Reply:
I also keep them separated is a good decision, in terms of performance we won't have all the affinity scores edges in a single invitation.